addons/mail/static/tests/discuss/message_pin/pinned_messages_tests.js

```diff
@@ -3,6 +3,7 @@
 import { click, start, startServer } from "@mail/../tests/helpers/test_utils";
 
 import { nextTick, patchWithCleanup } from "@web/../tests/helpers/utils";
+import { click as clickContains, contains } from "@web/../tests/utils";
```

Comment:
Probably it'll conflict in master, but just in case it doesn't: don't forget to rename to simply `click` ;)